src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1559,3 +1559,18 @@ def test_customtkinter(pyi_builder):
     pyi_builder.test_source("""
         import customtkinter
     """)
+
+
+@importorskip('pylibmagic')
+def test_pylibmagic(pyi_builder):
+    pyi_builder.test_source("""
+        import pylibmagic
+        import os
+
+        bundle_dir = os.path.normpath(sys._MEIPASS)
+        pylibmagic_data_path = f"{bundle_dir}/pylibmagic"
+        assert os.path.isfile(f"{bundle_dir}/libmagic.so.1", \
+            f"The libmagic compiled file was not collected to _MEIPASS!"
+        assert os.path.isfile(f"{bundle_dir}/magic.mgc", \
+            f"The magic.mgc file was not collected to _MEIPASS!"
```

Comment:
You're missing the `)` for each of the `isfile()` calls. That's what [this gobbledegook](https://github.com/pyinstaller/pyinstaller-hooks-contrib/actions/runs/4967899010/jobs/8890173480?pr=581#step:10:507) is trying to say.

Reply:
Thanks! When adding the line break, I mistakenly deleted the `).`

I've pushed the fix.